lib/tasks/inquisition.rake

```diff
@@ -3,23 +3,63 @@ require 'traceroute'
 namespace :inquisition do
   task :create_configs do
     configs = [
-      { gem: 'Overcommit', file: '.overcommit.yml' },
-      { gem: 'RSpec',      file: '.rspec' },
-      { gem: 'Rails ERD',  file: '.erdconfig' },
-      { gem: 'Rubocop',    file: '.rubocop.yml' },
-      { gem: 'Reek',       file: '.reek.yml' },
-      { gem: 'Fasterer',   file: '.fasterer.yml' }
+      { analyzer: 'Overcommit', file: '.overcommit.yml' },
+      { analyzer: 'RSpec',      file: '.rspec' },
+      { analyzer: 'Rails ERD',  file: '.erdconfig' },
+      { analyzer: 'Rubocop',    file: '.rubocop.yml' },
+      { analyzer: 'Reek',       file: '.reek.yml' },
+      { analyzer: 'Fasterer',   file: '.fasterer.yml' },
+      { analyzer: 'ESLint',     file: '.eslintrc.json' },
+      { analyzer: 'Stylelint',  file: '.stylelintrc.json' }
     ]
 
     configs.each do |config|
-      print "Creating config for #{config[:gem]}... "
+      file_name_without_extension = config[:file][/^\.\w+/]
+
+      # Find existing configs (for example: .reek.yml, .rspec) recursively
+      if Dir.glob("**/#{file_name_without_extension}{.*,}").any?
+        puts "Use existing config for #{config[:analyzer]}. "
+      else
+        print "Creating config for #{config[:analyzer]}... "
+        cp File.join(File.join(File.dirname(__dir__), 'configs'), config[:file]), Dir.pwd, preserve: true, verbose: false
+        puts 'Done!'.green
+      end
+    end
+
+    mkdir_p File.join(Dir.pwd, 'doc'), verbose: false
+  end
 
-      cp File.join(File.join(File.dirname(__dir__), 'configs'), config[:file]), Dir.pwd, preserve: true, verbose: false
+  task :install_eslint do
+    puts "Installing eslint locally (without configuration)..."
 
-      puts 'Done!'.green
+    system 'sudo npm install eslint --save-dev'
+
+    # You can also create package.json and configure .eslintrc from scratch.
+    # Uncomment following lines for this:
+    # system 'npm init -y'
+    # system 'sudo ./node_modules/.bin/eslint --init'
+
+    box 'Important!', color: :red do
+      "You need to create package.json file with all required dependencies for ESLint by yourself."
```

Comment:
It would be great to print the command for creating a default `package.json` or ask the question smth like "Would you like to create a default configuration?". What do you think?

Reply:
@timlar, it would be nice addition. I'll implement this